proto/wasm/v1/scalars.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package wasm.v1;
+
+import "cosmos_proto/cosmos.proto";
+
+option (cosmos_proto.declare_scalar) = {
+    name: "RawContractMessage",
+    description: "A type representing a decimal number",
+    field_type: [SCALAR_TYPE_BYTES],
+    // Optionally we can add these 2 fields:
+
+    // Proposal 1:
+    gogo_customtype: "RawContractMessage", // This makes sure all fields that have `cosmos.scalar = "RawContractMessage"` also have `gogoproto.customtype = "RawContractMessage"`.
+    // TODO is this useful?
+    
+    // Proposal 2: (similar to Dan's `hint` idea)
+    amino_encoding: "Use base64 to encode this field" // Some human-readable string on how to encode this field.
+    // TODO Or do we prefer machine-readable? e.g. `base64`
```

Comment:
this is cool! would be great to have 

`amino_encoding: "base64"` to we can just parse it directly.

perhaps we can add a description field or the dev can comment to help add context